sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducer/EventHubProducerLiveTests.cs

```diff
@@ -968,30 +968,6 @@ await using (EventHubProducer producer = client.CreateProducer())
             }
         }
 
-        /// <summary>
-        ///   Verifies that the <see cref="EventHubProducer" /> is able to
-        ///   connect to the Event Hubs service and perform operations.
-        /// </summary>
-        ///
-        [Test]
-        [Ignore("Expected behavior currently under discussion")]
-        public async Task ProducerCanSendWhenClientIsClosed()
-        {
-            await using (EventHubScope scope = await EventHubScope.CreateAsync(1))
-            {
-                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
-
-                await using (var client = new EventHubClient(connectionString))
-                await using (EventHubProducer producer = client.CreateProducer())
-                {
-                    client.Close();
-
-                    var events = new EventData(Encoding.UTF8.GetBytes("Do not delete me!"));
-                    Assert.That(async () => await producer.SendAsync(events), Throws.Nothing);
-                }
-            }
-        }
-
```

Comment:
Per discussion, the service team requests that we have a 1:1 relationship between the client and AMQP connections.   The client owns the connection, so when it is closed, any child consumer/producer will no longer be able to operate, by design.